plugins/auth0/pkg/fetch/fetch.go

```diff
@@ -141,6 +119,43 @@ func (f *Fetcher) fetchUsers(ctx context.Context, outputWriter *js.JSONArrayWrit
 	return nil
 }
 
+// return a object map to output or a boolean to skip current user.
+func (f *Fetcher) buildOutputObjects(ctx context.Context, user *management.User) (map[string]any, error) {
+	var obj map[string]any
+
+	res, err := user.MarshalJSON()
+	if err != nil {
+		return nil, err
+	}
+
+	if err := json.Unmarshal(res, &obj); err != nil {
+		return nil, err
+	}
+
+	obj["email_verified"] = user.GetEmailVerified()
```

Comment:
Why does "email_verified" need to be added explicitly? Wouldn't it be part of the map unmarshaled above (in line 131)?